### PR TITLE
Automatically find the most common BLAS dlls in Windows

### PR DESCRIPTION
**Pull request by AngelEzquerra**
This uses the fact that dynlib supports patterns to look for the most common BLAS dll names when loading the BLAS library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Only a subset of BLAS is available under `nimblas`, with more operations added
 on necessity.
 
 For a higher-level linear algebra library based on this, check out
-[neo](http://unicredit.github.io/neo/).
```
Comment (AngelEzquerra) [inline README.md:17]: This link was not working so I thought that Neo was discontinued. I now realize that it still exists. That being said I think it probably makes more sense to link to Arraymancer instead (or to both Arraymancer and Neo), given that Arraymancer is the most popular nim tensor library.

Comment (Vindaar) [reply README.md:17]: We should link to both.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Only a subset of BLAS is available under `nimblas`, with more operations added
...
+[arraymancer](https://github.com/mratsim/Arraymancer).
```
Comment (Vindaar) [inline README.md]:
```suggestion
- [neo](https://andreaferretti.github.io/neo/)
- [arraymancer](https://github.com/mratsim/Arraymancer).
```

**Comment by Vindaar** [comment]
Thanks a lot, this is really great!

**Pull request merged**